public/example.protocol.js

```diff
@@ -78,7 +78,8 @@ return {
       {
         "id": "namegen1",
         "type": "NameGenerator",
-        "title": "Name Generator Title",
+        "image": "menu-ng",
```

Comment:
I think the spec requires us to be able to specify it, so this seems an okay place to put it. Perhaps we should call it `icon` though?